devel/creduce/Portfile

```diff
@@ -50,23 +76,20 @@ checksums               rmd160  183be95aec282812724ec66204952ad33dbc923b \
 
 compiler.cxx_standard   2011
 
-if {${os.platform} eq "darwin" && ${os.major} <= 9} {
-    known_fail          yes
-    pre-fetch {
-        ui_error "${name} is only supported on Mac OS X 10.6 or newer."
-        return -code error "unsupported Mac OS X version"
+platform darwin {
+    if {${os.major} <= 9} {
+        known_fail          yes
+        pre-fetch {
+            ui_error "${name} is only supported on Mac OS X 10.6 or newer."
+            return -code error "unsupported Mac OS X version"
+        }
+    } elseif {${os.major} <= 12} {
+        # forcing libc++ works on these platforms
+        depends_lib-append      port:libcxx
+        configure.cxx_stdlib    libc++
```

Comment:
Is forcing use of `libc++` still necessary, particularly for 10.7 and 10.8?

Reply:
> Is forcing use of `libc++` still necessary, particularly for 10.7 and 10.8?

Admitedly my knowledge is still a bit light in this area though... and could use some guidance (and tutelage) relative to this...